### DISCUSSION TITLE
Broken themes page ohmyposh.dev/docs/themes?

### DISCUSSION THREAD

**Discussion by agenteo** [JanDeDobbeleer/oh-my-posh#4491] [General]
Hey there,

I am installing omp and I think the themes page is broken: https://ohmyposh.dev/docs/themes

Screenshot ⏬

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/16841/2f42ff1b-d25f-46a9-92eb-03cd7962cf55)

**Comment by JanDeDobbeleer** [comment]
This is related to the migration to Docusaurus V3 it seems. Fix in #4492
